src/providers/provider_dota.ts

```diff
@@ -71,6 +69,8 @@ export default class DotaProvider extends Provider {
     const patchList: string[] = [];
     const $ = pageDoc;
 
+    // This has to be a named function to set new `this` scope
```

Comment:
I'd really like us to change this whole thing, the syntax is completely different than what we have everywhere else...

Reply:
Let's create an issue about this and tackle it separately

Reply:
I moved this to issue #196.